src/discordMessenger.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+Connects to a Discord webhook and sends article to corresponding webhook.
+*/
+
+class discordMessenger
+{
+    private $url;
+
+    /*
+    Given a webhook URL this constructor sets the URL to the given webhook. 
+    */
+    public function __construct($url)
+    {
+        $this->url = $url;
+    }
+
+    /*
+    Encodes the article into JSON and POSTS it to the Discord webhook.
+    */
+    public function sendArticle($article)
+    {
+
+        $timestamp = date("c", strtotime("now"));
+
+        $json_data = json_encode([
```

Comment:
This seems to be the only variable in the code base with `snake_case` instead of `lowerCamelCase`

Reply:
thanks_for_telling_me, issueShouldBeResolvedNow 4fdd0075f1cda58b0eb490dcdf6800e4b6c24fd9